cms/static/js/views/list_item.js

```diff
@@ -29,6 +29,7 @@ define([
             var index = this.model.collection.indexOf(this.model);
 
             return [
+                'wrapper-collection',
                 'group-configuration',
```

Comment:
Should be able to remove group-configuration items... hopefully. If not, things will need to be pushed up.